spec/system/task_spec.rb

```diff
@@ -86,7 +88,7 @@
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
         project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        task = FactoryBot.create(:task, :due_yesterday, project_id: project.id) # traitを利用して再利用性を高める
```

Comment:
`due_yesterday`というtrait名では「期限が昨日であること」が読み取れますが、タスクのステータスも完了になっています。
yesterdayであることを特段強調する必要もないと思うので今回は`done`だけでも問題ないかと個人的には思います。
```suggestion
        task = FactoryBot.create(:task, :done) # traitを利用して再利用性を高める
```

Reply:
commit IDは、 c3980cb  になります。